pkg/services/smtp/smtp_config.go

```diff
@@ -23,7 +24,7 @@ type Config struct {
 	Subject     string    `desc:"The subject of the sent mail" key:"subject,title" default:"Shoutrrr Notification"`
 	Auth        authType  `desc:"SMTP authentication method" key:"auth" default:"Unknown"`
 	Encryption  encMethod `desc:"Encryption method" default:"Auto" key:"encryption"`
-	UseStartTLS bool      `desc:"Whether to use StartTLS encryption" default:"Yes" key:"starttls"`
+	UseStartTLS bool      `desc:"Whether to use StartTLS encryption" default:"Yes" key:"usestarttls"`
```

Comment:
We don't want to make a breaking change, but demoting `starttls` to an alias is fine.
```suggestion
	UseStartTLS bool      `desc:"Whether to use StartTLS encryption" default:"Yes" key:"usestarttls,starttls"`
```

Reply:
Alright, wasn't sure whether you had a way of making it an alias that wouldn't show in the docs for if you ever wanted to remove this alias

Reply:
We might make some breaking changes in a `1.0` release, but until then I see no reason to remove an alias. Users generally just configure once and then expect it to continue working. So that seems like the highest priority.